tests/e2e/cucumber/features/smoke/urlJourneys.feature

```diff
@@ -6,6 +6,9 @@ Feature: web can be navigated through urls
       | id    |
       | Alice |
       | Brian |
+    And "Admin" assigns following roles to the users using API
+      | id    | role        |
+      | Alice | Space Admin |
```

Comment:
:+1:
Ah, so the API steps for `Admin` was mixed up with `Alice`.
NOTE: In e2e, we do API requests before login